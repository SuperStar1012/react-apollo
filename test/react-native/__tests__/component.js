// __tests__/Intro-test.js
import 'react-native';
import React from 'react';
import App from '../component';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

describe('App', () => {

  it('renders correctly', () => {
    const tree = renderer.create(
      <App />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });

});
