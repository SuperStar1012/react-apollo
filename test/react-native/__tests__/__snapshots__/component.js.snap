exports[`App renders correctly 1`] = `
<Text>
  Loading...
</Text>
`;
